scripts/release.sh

```diff
@@ -119,14 +119,14 @@ cd $SCRIPTDIR/..
 
 if [ -z "$IMAGES" ]; then
   if [ -z "$DEBUG" ]; then
-    IMAGES="agents rest"
+    IMAGES="core jsongrpc rest"
   else
-    IMAGES="agents-dev rest-dev"
+    IMAGES="core-dev jsongrpc-dev rest-dev"
```

Comment:
btw you can use instead  `agents-dev.core` and `agents-dev.jsongrpc` and leave the nix file as is.

Reply:
Thanks, that's good to know. I'll leave it as it is for now because I'll need to fix up the release script to work with the changed names. I'll raise a work item for this and circle back to it.